src/test/java/org/opentripplanner/ConstantsForTests.java

```diff
@@ -72,6 +72,9 @@ public class ConstantsForTests {
   public static final String FARE_COMPONENT_GTFS =
     "src/test/resources/gtfs/farecomponents.gtfs.zip";
 
+  public static final String SHAPE_DIST_GTFS =
+    "src/test/resources/gtfs/shape_dist_traveled_gtfs.zip";
```

Comment:
```suggestion
    "src/test/resources/gtfs/shape_dist_traveled/";
```
If you unzip the file then it should just work with a path.

Reply:
See line 70 in the same file.

Reply:
Right, it is obviously much better solution.